### PR TITLE
elastic: added more backend tests, copied from the frontend code

### PR DESCRIPTION
**Pull request by gabor**
the tests were copied from https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/ElasticResponse.test.ts

some parts are failing, those were commented out with a `FIXME` comment.

the plan is to keep improving the go code until the tests pass.

### CODE REVIEW

File: pkg/tsdb/elasticsearch/response_parser_frontend_test.go [hunk 1]
```diff
@@ -0,0 +1,1686 @@
...
+	requireTimeValue(t, 1000, frame, 0)
+	requireNumberValue(t, 10, frame, 0)
+}
+
+func TestSimpleQueryCountAndAvg(t *testing.T) {
+	query := []byte(`
+	[
+		{
+			"refId": "A",
+			"timeField": "t",
```
Comment (ivanahuckova) [inline pkg/tsdb/elasticsearch/response_parser_frontend_test.go:301]: I'd personally use `"timeField": "@timestamp",` everywhere as we use in all other tests, but that is just nit pick.

File: pkg/tsdb/elasticsearch/response_parser_frontend_test.go [hunk 2]
```diff
@@ -0,0 +1,1686 @@
...
+				},
+				{
+				  "_id": "kdospaidopa",
+				  "_type": "_doc",
+				  "_index": "mock-index",
+				  "_source": {
+					"@timestamp": "2019-06-24T09:52:19.765Z",
+					"host": "dsalkdakdop",
+					"number": 2,
+					"message": "hello, i am also message",
```
Comment (ivanahuckova) [inline pkg/tsdb/elasticsearch/response_parser_frontend_test.go:1476]: `hello, i am also message`
😂

**Review by ivanahuckova — APPROVED** [review]
👍

File: pkg/tsdb/elasticsearch/response_parser_frontend_test.go (hunk 1 above)
Comment (gabor) [reply pkg/tsdb/elasticsearch/response_parser_frontend_test.go:301]: i see your point, but i'd argue we should use some other value than `@timestamp`, the reason is, we default to `@timestamp` when there is no time-field in the datasource-config, so if we want to make sure a customized time-field-value works, we should find a non-default value. anyway, can talk talk about later 👍 , i can change it, it's not a big deal.

(or we can just drop this field completely.. probably).

**Pull request merged**